CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to
 
 ## Added
 
+- Add orders with signed and on unsigned contract to demo_dev management
+  command.
```

Comment:
This change is only for dev purpose, so there is no need to add an entry into the CHANGELOG for that